main.tf

```diff
@@ -0,0 +1,34 @@
+terraform {
+  required_providers {
+    fmc = {
+      source = "CiscoDevNet/fmc"
+      version = "0.2.1"
+    }
+  }
+}
+
+provider "fmc" {
+    fmc_username = "admin"
+    fmc_password = "Cisco@123"
+    fmc_host = "13.235.51.192"
```

Comment:
Do you want to commit public IPs with username/passwords? Should this main.tf even be added to this repo?

Reply:
Removed